src/settings/StaffSlips/components/ViewSections/StaffSlipTemplateContentSection/StaffSlipTemplateContentSection.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+  injectIntl,
+} from 'react-intl';
+import HtmlToReact, { Parser } from 'html-to-react';
+
+import {
+  Button,
+  Col,
+  Row,
+} from '@folio/stripes/components';
+import { PreviewModal, tokensReducer } from '@folio/stripes-template-editor';
+import css from '../../../StaffSlipDetail.css';
```

Comment:
Could you create a separate CSS file near `StaffSlipTemplateContentSection.js` file and remove the original one?